Formula/veloren.rb

```diff
@@ -0,0 +1,61 @@
+class Veloren < Formula
+  desc "Multiplayer voxel RPG"
+  homepage "https://veloren.net"
+  url "https://gitlab.com/veloren/veloren.git",
+    tag:      "v0.13.0",
+    revision: "34ae5b9df73f367fa34350337ebba71711f48b86"
+  license "GPL-3.0-or-later"
+
+  depends_on "cmake" => :build
+  depends_on "git-lfs" => :build
+  depends_on "ninja" => :build
+  depends_on "rustup-init" => :build
+
+  on_linux do
+    depends_on "pkg-config" => :build
+    depends_on "python@3.10" => :build
+    depends_on "alsa-lib"
+    depends_on "libxcb"
+    depends_on "libxkbcommon"
+    depends_on "systemd" # for libudev
+  end
+
+  def install
+    system "git", "lfs", "install", "--local"
+    system "git", "lfs", "pull"
+
+    # make sure the assets have been downloaded correctly with git-lfs
+    unless `file assets/voxygen/background/bg_main.jpg`.match?(/JPEG image/i)
+      odie "assets have not been downloaded correctly"
```

Comment:
We should use `resource` blocks to download these assets.

Reply:
fixed, now uses a `resource` block